templates/look-and-feel/components/fields.njk

```diff
@@ -85,6 +88,12 @@
            value="{{ option_value }}">
     <label for="{{ field.id }}-{{ option_value }}">
       {{ option.label }}
+      {% if option_description %}
+          <br>
+          <span>
+            {{ option.description | safe }}
```

Comment:
safe shouldn't be used by default, it opens up xss exploits, it should only be used for static content and only where needed

Reply:
👍